apps/tasks/lib/components/task_status_pill.dart

```diff
@@ -57,11 +57,18 @@ class TaskStatusPill extends StatelessWidget {
             shape: shape ?? const RoundedRectangleBorder(),
           ),
           child: Row(children: [
-            Circle(color: dotColor ?? Colors.black),
-            const SizedBox(width: distanceTiny),
+            Circle(
+              color: dotColor ?? Colors.black,
+              diameter: 7.5,
+            ),
+            const SizedBox(width: 4),
```

Comment:
Make this an attribute of the widget or use a predefined style constant like `distanceTiny`